src/pages/Browse/index.spec.tsx

```diff
@@ -181,4 +353,57 @@ describe(__filename, () => {
       path,
     });
   });
+
+  it('renders a global LinterMessage', () => {
+    const file = 'lib/react.js';
+    const externalMessage = globalExternalMessage({
+      file,
+      message: 'This is a known third party library',
+    });
+
+    const root = renderWithMessages({
+      file,
+      externalMessages: [externalMessage],
+    });
+
+    const message = root.find(LinterMessage);
+    expect(message).toHaveLength(1);
+    expect(message).toHaveProp(
+      'message',
+      createInternalMessage(externalMessage),
+    );
+  });
+
+  it('renders all global LinterMessage components', () => {
+    const firstUid = 'first-uid';
+    const secondUid = 'second-uid';
+
+    const file = 'lib/react.js';
+    const root = renderWithMessages({
+      file,
+      externalMessages: [
+        globalExternalMessage({ uid: firstUid, file }),
+        globalExternalMessage({ uid: secondUid, file }),
+      ],
+    });
+
+    const message = root.find(LinterMessage);
+    expect(message).toHaveLength(2);
+    expect(message.at(0).prop('message')).toMatchObject({ uid: firstUid });
+    expect(message.at(1).prop('message')).toMatchObject({ uid: secondUid });
+  });
+
+  it('ignores global messages for the wrong version', () => {
```

Comment:
Would it be worth also having a test where the version matches but we load external messages for `pathA`, and then render with `pathB`, and we make sure the messages are not rendered?

Reply:
Yes, good idea. That's important logic to cover.